apps/src/code-studio/pd/workshop_dashboard/components/survey_results/single_choice_responses.jsx

```diff
@@ -0,0 +1,70 @@
+import React, {PropTypes} from 'react';
+import _ from 'lodash';
+import {Panel} from 'react-bootstrap';
+
+export default class SingleChoiceResponses extends React.Component {
+  static propTypes = {
+    question: PropTypes.string.isRequired,
+    answers: PropTypes.object.isRequired,
+    answerType: PropTypes.string.isRequired,
+    possibleAnswers: PropTypes.array.isRequired,
+    otherAnswers: PropTypes.array
```

Comment:
What is `otherAnswers`? Is that a collection of all filled-in "other" responses to this question? Please ass some description here for the component and its props.

Reply:
+1 a brief explanation of `answers` as opposed to `otherAnswers`